llama-index-core/llama_index/core/indices/struct_store/sql_query.py

```diff
@@ -351,10 +351,11 @@ def __init__(
 
         self._synthesize_response = synthesize_response
         self._verbose = verbose
+        self._streaming = kwargs["streaming"] if "streaming" in kwargs else False
```

Comment:
I think it might be better to include `streaming` as a param in `__init__()` instead and give it a default value of `False` (i.e. treat it like `verbose`).

CC: @logan-markewich

Reply:
I agree, I would go with that 👍🏻

Reply:
ok, i have done this change, please review

Reply:
Thanks!